examples/EdgeCaseSaliency.ipynb

```diff
@@ -0,0 +1,1318 @@
pwStPHysDTHsSMoGQelYl/qWEmQYj8tgpLKe/ORW1OCY9w/GuH8UgpNG3zfvDg4OOlRFLmsy2246FaWVmnO1w67ckjjuRSFo2yD/AN9VUsNqaosUbCaNxgt69+lbs+mIcOkZ2/xYOCKmcUnZCje1zGSZRIPLySK9Ct3ivdOsZJNrBo9pLevyj+dcWbYRyfN8vHYdqv6fqIjK2DNiNiSp9Dj/ABxSTWw3foS3fh6GeZjbSODuIIVCcH0pIfCOTl/Pb2Yqv+NdbbMqpKGYDa5JHoCanOMYwSDT5mth7q5zcHhG1U/Mif8AAnZv5Yq3/wAI7p8CbnO0D+6oX/69bW3IGP1psqCWPZIoZTReTD3TPXSLQQbreWcjrgTMOad9hiv9PXTg3kzRg/Zp255JztY9wSatrFHBESB8gHPNV4FSW3Dneu/nax5HtxVxc07kNwscjYARBzuLMzfMSc81qIwP0qzPo1skObTKMgztyTu/Os9QSpApzcm+aQ4yi9EKt1aXzG2Dq5ZN2PbOKfbaXDGSkhEkecgMOlZmj6eLObeQxkKFS57/ADE4rfU5qGUV9as5bm3jW2XcqAA4++P/AK1Y2nzq+pRQT26Z85Y3DDBAJ64z2/rXTqzLyh5quLGyLNJ9n/euSzAc5P41nGChsVzJqzRpw67dLf31g8ZngEphRs5KZAAznr169a1Y1h1S4TbwMFWA6qcc1zOnzW1jqCQFHSWWVXGctuwCck/UCs641SSxmmuo5PL27nGe4yR0/wCA1tVUVBNF0Pia2OzS6urDzoUcvCh2h8fd9x6fTpWtYait9iNxiQDOV6Ecfl1rm9D8SW+owXMUyNHcSfxNwGJHb1q4ttPaQJeW8pVvT15I4/wrGLaNZQjNNor6Rc6nFq+owXrtJbG4Y2zPjJTPGCP5Vk6xcyWN2kbRhpJpPXoCev61Zm1Lz4haRlo7kXAL54HOD8tLqLCW/diASvGal257owe1mc7fX032xoll2gcfKo9PU1nTzEn5md/99ia1jpIkneaSU5ZicCpk061Q58sMR3bmqUkjBwkzKsR5l5GoA2rycDHSth4440faoUyHBI7k08BE4AVR7cVTu7giWNIwXxlmx29P50r3ZcY2J92ST2ppZR1NUpriRIXfBAVSelU7W4luLd55CAgzgfSlY0uaUl0i55yfaqss8sinb8q4PNUtMCSl5mZiScDf2qxqMpFlIkWN7DA/GqtZ2FfQ5RG+03xcEAkk8963bdEC/NwR2qtbWARBuGXBzk+tXWjcx7Ao+prZo553lsVr7IiCKOTzx79P0qlCnkw4YjcTzV6W2mkkYlvl7euKrukEEiI4c7urYzj8KEPldiuxL8KCarDzTdbNoGRySM1uCNSowoxUN1avbxmZShRs/MvByPUdaDSFNMt22oLFCqSqQQMZFaNjLEAZRHFIGP8AEo/n1FcYZwD97caltbi737oCUJIAxzuzSOlT0tLU9S06+W6nS0j/ANY/AjY5Fb/iLVbjw9pltFp8UG8/KPMOAAPQVzvgDS7qbUJdRvYlUwKY0I/iY9/wH86b45S7utRWWSBxawfdJHB9/Xr39KS13MJWjJ2R0Xh7xRJqd2LG9gEVzs3AgEBsda0tb1A2kJdQGZOFU9C1Y3gnTWtNFF/dZeeUlo93JVenH1rA8U+IJv7eTT7XBES75nIztJ7fXoPxosK9zeGoaTrb+Xqmn/Z7gLnzo/8AEc/nmqN34QaWE3Gk3cV5DnoGGfpkcH9K5O7uJrBj5kz+dND87EfKMngZrKtNTl0273W1zPaSKcGWFsKx75HQ0KJcakom9cWdxbTGKeJ43HZxio38vyXibOGGM9x7irj+I5dXRIdVRZSOYrq1GWAP95O4+lOvtA1G2BKxCePs0RyfxHUflU+p1QqRmrM5+SGRflWaIg924NNj0/cCfkkI/u4P8qdcRyRMRIjofRwR/OprWIou7oTRypbCdLm2ZSceZMIU7cHHb1qyYyI9qHacYHGcVdypYsyKzHq2MH86gkQ5JiAP+yxwfz6VNmiasJaJLRGPLa3zMwaSNlPTjBNe8fBWLyfBl0uAD9vcnH/XOOvHHyiL5q7GPY817R8Hv+RSu/T7e/8A6Ljpx3MG3ax6FRRRWhBm+IQD4a1UHp9jm/8AQDXgDQogaVgdo9a9/wDEIz4a1UAZzZzf+gGvnzypbh9ikEDk88Cs6gDixkH7tQifmTTRvXAViGPTmpDbPHHtV/3mc44GBUf2dY42d28yRjhR2H4VmMRlaPCBssT+A96upF5EYjI+c/M/17D8v51BbxZYlgWRPmc46eg/Gp3c/eJ+Y8nFb0Y68xjVl0M7VL2OztyWIL/wj1rkVdpJScksSSTirmuXPnaiyDJVBjHvVaEc4LOPTAqqjMokyRKxG5yGHXOKtLGIFBA3sR61GAGbapOW4weBU6WjOdqyKD3J6Vg2ax20I3LIvzKcEZYqOtPikzMwj3BAhzgE9qkd1gQISrbMZb1qAXSvgJHt+Ug7e/FJO62IWobWuQNkZYBu46dKuTW/nwouVaUYBUDOaSCNhacEqrkEgcHH86uWkckNpLMMbcjaPboacexajYymSbIiJ2qDhR/hUF0jQIzucuOma0ltz9oiVjk7s8VV1CJJ7lovmBJAwOc/4U0OSsUIZJ/LJB+duy9BV20NxM2x1UsejHjFXbKzNuokbau7+A8tj8elWLkIsO2N9iAndt9+1ctSrrZGUmc/KCkhRjt5x8vOaqPI6srbC+0YBY8j8q1ZrOFYl5OCclQcnFLDaQyFV8v5uw254+pNWqisK5jo8jEtIxGBwPQVsaZJhyEZckZGe/8AnNK9tZRSGMq8ZZuW6nH51NDbpHMpjjYIBjMnf3FUqqZUZK50kHzRmnpgNiorZwVxnqKfn5664vQ2kWww4xTlPzGolPFSA/MOe1UIlqXh4iDUQNSR9xQ9iVuZOqaRDfRFmjVpF+6SKq2Wlm1GY4kxnj1rf4xg1GFP15rB
```

Comment:
We will likely want provide a "saved" notebook with this set to False in order to add the lease amount of "distraction" (read: logging statements) as possible.